packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.spec.tsx

```diff
@@ -126,7 +126,11 @@ describe('rendering', () => {
       expect(rendered.queryByText('Not found')).toBeInTheDocument();
     });
   });
-  describe('when request fails', () => {
+  // TODO: the test is a bit flaky, where the `onError` callback does not get called
+  // consistently. This seems to be an internal issue with the apollo client.
+  // For now we can skip the test and maybe check again if it works in future versions
+  // of apollo.
+  describe.skip('when request fails', () => {
```

Comment:
Should we then also change the implementation? I think we will actually never change this back knowing how we handled stuff like this in the past :)

Reply:
Then lets remove it.